kube_dns/assets/service_checks.json

```
@@ -1 +1,14 @@
-[]
+[
+    {
+        "agent_version": "6.0.0",
```

```suggestion
        "agent_version": "7.42.0",
```

For tracking purposes these should point to the agent version where these were introduced